### PR TITLE
Heart of the alien recipe

### PR DESCRIPTION
**Pull request by EppMouse**

### CODE REVIEW

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe [hunk 1]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
```
Comment (humdingerb) [inline games-action/heartofthealien/heartofthealien-1.2.2.recipe]: How about a bit more wordy DESCRIPTION? See the first 2 paragraphs of "About" at http://hota.sourceforge.net/
Hmm... is the 2nd paragraph correct?

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+INSTALL()
+{
+	install -d $appsDir/heartofthealien $docDir
+	install -t $appsDir/heartofthealien src/alien
+	install -t $appsDir/heartofthealien "Heart Of The Alien (U)".*
+	install -t $appsDir/heartofthealien rungame.sh
+	install -t $docDir doc/*
+	addAppDeskbarSymlink $appsDir/heartofthealien/rungame.sh heartofthealien
```
Comment (humdingerb) [inline games-action/heartofthealien/heartofthealien-1.2.2.recipe]: The link in the Deskbar would look nicer as "Heart\ of\ the\ Alien"

**Comment by EppMouse** [comment]
@extrowerk , About my INSTALL:
1. Create on my PC with Haiku OS folder:/boot/home/haikuports/game-action/heartofthealien
2. Copy my heartofthealien-1.2.2.recipe to folder:/boot/home/haikuports/game-action/heartofthealien
3. Open Terminal
4. hp heartofthealien
5. Open /boot/home/haikuports/packages/heartofthealien-1.2.2.hpkg
6. Install package
7. Open Applications folder
8. Run heartofthealien

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe (hunk 2 above)
Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Edited.

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe (hunk 1 above)
Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Edited

**Comment by EppMouse** [comment]
@humdingerb , I make changes.

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe [hunk 3]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times, Another World. Developed back in 1994 by Interplay exclusively for the Sega CD console. This is an open-source project which aims to bring this highly-addictive game to all modern consoles and computer platforms."
```
Comment (humdingerb) [inline games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Remove "the" after the "undoubtably". And: "'Another World', developed ..."
Also, mind the 80 char limit and insert a "` \`" and a line-break where needed.

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Edited.

Comment (humdingerb) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: I'm sorry, I was unclear. I meant: "add inverted commans around 'Another World', and a comma after it, and continue with a lower-case "developed".
Not remove all the text after "Another World"... :)

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Edited. Please check again.

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe [hunk 4]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times,\ „Another World“, developed back in 1994 by Interplay exclusively\ for the Sega CD console. This is an open-source project\ which aims to bring this highly-addictive game to all modern\ consoles and computer platforms."
```
Comment (humdingerb) [inline games-action/heartofthealien/heartofthealien-1.2.2.recipe]: You're missing the line-breaks after the " \"... :)
I suggest building the package once more with your changes, open it in HaikuDepot and see if the description looks good.

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: sc

Nice :)

File: games-action/heartofthealien/heartofthealien-1.2.2.recipe [hunk 5]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times, 'Another World', developed back in 1994 by Interplay exclusively for the Sega CD console. This is an open-source project which aims to bring this highly-addictive game to all modern consoles and computer platforms."
```
Comment (humdingerb) [inline games-action/heartofthealien/heartofthealien-1.2.2.recipe]: It's still one long line. You need to add a line break at the limit of 80 letters.
You have to escape that linebreak with a "` \`". Mind the space in front of the slash. If there isn't one, words will appear written together when viewed in HaikuDepot. That's why I suggested to re-build and see if it's OK.
But first you have to insert those linebreaks.

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: 0X0D
0X0A
INSERTED.

Comment (humdingerb) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Still missing the escaping "\" before the line breaks.

Comment (humdingerb) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: that should say "\\"

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Now correctly?

Comment (humdingerb) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Almost. Should look like this:

DESCRIPTION="Heart of The Alien is the sequel for the undoubtably \\
the bestadventure of all times, 'Another World',  developed back \\
in 1994 by Interplay exclusively for the Sega CD console.

This is an open-source project which aims to bring this \\
highly-addictive game to all modern consoles and computer platforms."

Comment (EppMouse) [reply games-action/heartofthealien/heartofthealien-1.2.2.recipe]: Edited.

**Comment by humdingerb** [comment]
Thanks. That was painful, eh?  :) :)

**Comment by EppMouse** [comment]
But now how nice

**Comment by EppMouse** [comment]
х)

**Pull request merged**